project-management-application-app/src/test/java/com/newklearz/controllers/RegistrationControllerIT.java

```diff
@@ -0,0 +1,25 @@
+package com.newklearz.controllers;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.newklearz.ProjectManagementApplication;
+import com.newklearz.DTO.AppUserRole;
+import com.newklearz.DTO.UsersDTO;
+import com.newklearz.adapters.UserAdapter;
+import com.newklearz.repository.users.Users;
+
+@SpringBootTest(classes = ProjectManagementApplication.class)
```

Comment:
Can we create a "SpringBootTestEnvironment" class and add all annotation and services there ?